app/views/govuk_publishing_components/components/docs/hint.yml

```diff
@@ -25,3 +26,9 @@ examples:
     data:
       text: "العربيَّة"
       right_to_left: true
+  with_js_class:
```

Comment:
You're very welcome to add an explicit documentation of this feature, but just to let you know that the options for the component wrapper are automatically added to the top of this guide page when `uses_component_wrapper_helper` is included.

Reply:
That's a good shout. I'll remove it. I added a test for it already so there's something to stop a regression.